query-node/mappings/src/proposals.ts

```diff
@@ -305,6 +307,12 @@ async function parseProposalDetails(
     const asPayload = unwrap(specificDetails.payload)?.objectCreationParams
     details.payloadHash = asPayload && bytesToString(asPayload.ipfsContentId)
 
+    return details
+  } else if ((proposalDetails as RuntimeProposalDetails_V2002).isSetPalletFozenStatus) {
+    const details = new UpdatePalletFrozenStatusProposalDetails()
+    const specificDetails = (proposalDetails as RuntimeProposalDetails_V2002).asSetPalletFozenStatus
+    details.frozen = specificDetails[0].isTrue
+    details.pallet = specificDetails[1].type
```

Comment:
Perhaps even clearer code:
```suggestion
    const [frozen, pallet] = (proposalDetails as RuntimeProposalDetails_V2002).asSetPalletFozenStatus
    details.frozen = frozen.isTrue
    details.pallet = pallet.toString()
```